src/components/Tokens/index.tsx

```diff
@@ -7,25 +8,23 @@ import { displayMonetaryAmount } from '@/common/utils/utils';
 import ErrorFallback from '@/components/ErrorFallback';
 import { SELECT_VARIANTS, SelectVariants } from '@/components/Select';
 import {
-  CollateralToken,
   GOVERNANCE_TOKEN,
   GOVERNANCE_TOKEN_SYMBOL,
-  GovernanceToken,
   GovernanceTokenLogoIcon,
   RELAY_CHAIN_NATIVE_TOKEN,
   RELAY_CHAIN_NATIVE_TOKEN_SYMBOL,
+  RelayChainNativeToken,
   RelayChainNativeTokenLogoIcon,
   WRAPPED_TOKEN,
   WRAPPED_TOKEN_SYMBOL,
-  WrappedToken,
   WrappedTokenLogoIcon
 } from '@/config/relay-chains';
 import { useGovernanceTokenBalance } from '@/services/hooks/use-token-balance';
 
 import TokenSelector from './TokenSelector';
 
 interface TokenOption {
-  token: WrappedToken | CollateralToken | GovernanceToken;
+  token: WrappedCurrency | RelayChainNativeToken | GovernanceCurrency;
```

Comment:
@peterslany 
Good question. The other day I suggested using "token" instead of "currency". Actually, in the UI, we have been using "token" consistently. `WrappedCurrency` and `GovernanceCurrency` are from the lib while `RelayChainNativeToken` is from the UI. That's why there are mixtures of "token" and "currency".

I remember discussing it with @bvotteler but not sure about the context.

I think we should ask @bvotteler.